ReactSkia/components/RSkComponentScrollView.cpp

```diff
@@ -74,6 +74,65 @@ RnsShell::LayerInvalidateMask RSkComponentScrollView::updateComponentState(
   return RnsShell::LayerInvalidateNone;
 }
 
+void RSkComponentScrollView::handleCommand(std::string commandName,folly::dynamic args){
+  if(commandName == "scrollToEnd") {
+    if(args.size() != 1 ) {
+      RNS_LOG_ERROR("Command scrollToEnd received " << args.size() << " arguments,expected 1");
+      return;
+    }
+
+    RNS_LOG_DEBUG("handleCommand commandName[scrollToEnd] args[" << args[0] <<"]");
+
+    SkPoint lastScrollOffset = SkPoint::Make(0,0);
+    RnsShell::ScrollLayer* scrollLayer= SCROLL_LAYER_HANDLE;
+    if(isHorizontalScroll())
+      lastScrollOffset.fX = scrollLayer->getContentSize().width() - scrollLayer->getFrame().width();
+    else
+      lastScrollOffset.fY = scrollLayer->getContentSize().height() - scrollLayer->getFrame().height();
```

Comment:
Is there any chances that content size becomes smaller that frame size ? IN that case we will have negative value.

Reply:
Yes its possible. Have missed this use case. 
WIll address it.